conda_oci_mirror/oci_mirror.py

```diff
@@ -86,28 +88,40 @@ def reverse_tag_format(tag):
     return tag.replace("__p__", "+").replace("__e__", "!")
 
 
-def upload_conda_package(path_to_archive, host, channel, extra_tags=None):
+def upload_conda_package(path_to_archive, host, channel, annotations_key, oci, extra_tags=None):
     path_to_archive = pathlib.Path(path_to_archive)
     package_name = get_package_name(path_to_archive)
 
+    layers =[]
     with TemporaryDirectory() as upload_files_directory:
         shutil.copy(path_to_archive, upload_files_directory)
 
         prepare_metadata(path_to_archive, upload_files_directory)
 
+        fn = upload_files_directory + "/" + path_to_archive.name
+        md5_value = compute_md5(str(fn))
+        #_annotations_dict = {"org.conda.md5": md5_value}
+
+        _annotations_dict = {annotations_key: md5_value}
```

Comment:
you can hard code the `org.conda.md5` here since this is the `upload_conda_package` function.

Reply:
done